lib/intercept.js

```diff
@@ -123,6 +123,11 @@ function remove(interceptor) {
 }
 
 function removeAll() {
+  Object.keys(allInterceptors).forEach(function(key) {
+    allInterceptors[key].scopes.forEach(function(interceptor) {
```

Comment:
I'm not sure if I'm understanding this correctly but aren't the nested properties like `scopes` for example, guaranteed to be present? Don't see checks for these in any of the other functions.